lib/src/lib.rs

```diff
@@ -47,6 +48,8 @@ pub mod revset_graph_iterator;
 pub mod rewrite;
 pub mod settings;
 pub mod simple_op_store;
+#[allow(dead_code)]
```

Comment:
A brief comment justifying this (presumably due to generated code) wouldn't be amiss.

Reply:
It looks like it's actually not needed anymore (not sure if I just added calls to all the functions or if it changed with the upgrade to 0.17.0).